src/utils/editor/index.js

```diff
@@ -89,5 +138,12 @@ module.exports = {
   findNext,
   findPrev,
   removeOverlay,
-  SourceEditor
+  SourceEditor,
```

Comment:
I was hoping to make the index export all the associated utils and then only import from the dir.